### PR TITLE
Remove channel specific portion of legal text for P7

### PR DESCRIPTION
**Pull request by danielkocsan**

### CODE REVIEW

File: templates/banner.ejs [hunk 1]
```diff
@@ -3,9 +3,9 @@ var inner = '<div id="agttcnstbnnr"' +
...
+    '<span>Der Verein Arbeitsgemeinschaft Teletest (kurz AGTT; Details siehe agtt.at/hbb-Messung)<% if(!IS_PRO7){ %>, deren Mitglied <b><%-CHANNEL_NAME%></b> ist<%}%>, möchte das Nutzungsverhalten der Zuseher:innen erfassen, um dadurch den Mitgliedern der AGTT die Möglichkeit zu geben, deren TV Angebot stetig verbessern zu können. Dazu benötigt die AGTT Ihre Zustimmung, nach der ein Cookie auf Ihrem Gerät platziert wird, um folgende Informationen auslesen zu können:<br></br>' +
       'Geräte ID, IP-Adresse, System- bzw. Browserinformationen, Geräteinformationen (verwendet HbbTV-Version, TV-Hersteller, Übertragungsweg via Satellit oder Kabel, Geräteauflösung). ' +
-      'Durch Drücken der blauen Taste gelangen Sie zur Datenschutzerklärung, wo Sie auch den Status Ihrer Einwilligung verwalten können.</span></div>' +
+      'Nähere Informationen zum Datenschutz finden Sie in der HbbTV Applikation des Senders, wo Sie den Status Ihrer Einwilligung verwalten bzw. ändern..</span></div>' +
```
Comment (Mgutjahr) [inline templates/banner.ejs:8]:
```suggestion
      'Nähere Informationen zum Datenschutz finden Sie in der HbbTV Applikation des Senders, wo Sie den Status Ihrer Einwilligung verwalten bzw. ändern können.</span></div>' +
```

File: src/controller/util/getTemplateValues.ts [hunk 2]
```diff
@@ -9,8 +9,9 @@ import {
 } from "../../config";
 
 import { logger } from "../../util/logger";
+import { configuredCounterMetric, technicalAgeMetric } from "../../util/metrics";
 
-export const getTemplateValues = (req: Request): { [key: string]: any } => {
+export const getTemplateValues = (req: Request, type: string = "3rd-party"): { [key: string]: any } => {
```
Comment (Mgutjahr) [inline src/controller/util/getTemplateValues.ts:14]: I do not get the meaning of the `type` parameter? .. as far as I can see this is only used as an additional annotation for metrics. - But cannot we always set this to the same / hence use a constant?

Comment (Mgutjahr) [inline src/controller/util/getTemplateValues.ts:14]: I do not get the meaning of the `type` parameter? .. as far as I can see this is only used as an additional annotation for metrics. - But cannot we always set this to the same / hence use a constant?

**Review by Mgutjahr — APPROVED** [review]
Looks good, check out the comments pls

File: src/controller/util/getTemplateValues.ts (hunk 2 above)
Comment (danielkocsan) [reply src/controller/util/getTemplateValues.ts:14]: This is only to label the metrics whether the traffic comes through a device which supports iframes. 3rd-party(cookie) / iframe

Comment (danielkocsan) [reply src/controller/util/getTemplateValues.ts:14]: This is only to label the metrics whether the traffic comes through a device which supports iframes or not. 3rd-party(cookie) / iframe

File: templates/banner.ejs (hunk 1 above)
Comment (danielkocsan) [reply templates/banner.ejs]: Just removed the extra dot for now. Will double check with Nico.

**Pull request merged**